docs/playbook/getting-set-up/local-pyspark-environment.md

```diff
@@ -263,3 +263,17 @@ Make sure aliases for Python are on but not for the Python App installer.
 
 **Solution**: Download https://visualstudio.microsoft.com/visual-cpp-build-tools/ and install the 3 first tools
 
+### Issue: Various PySpark functions can no longer be found**: Mac
+
+**Solution**: Uninstall PySpark and reinstall. The latest version should generally be ok, but if in doubt reinstall the version you were using previously.
+
```

Comment:
I think we maybe want to be a bit prescriptive about this - possibly elsewhere and linked to? 
To align with the supported versions in Glue (https://docs.aws.amazon.com/glue/latest/dg/release-notes.html)
Tricky while we're still mainly using Glue 2, but plan to move to Glue 3 and there are things that work in Spark 2.4.3 that are broken in 3.1.1